index.html

```diff
@@ -1,65 +1,247 @@
+
+                        <li class="project">
+                            <h3>Profesional Art<br>Printing Data</h3>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                            <ul class="build-with">
+                                <li>html</li>
+                                <li>bootstrap</li>
+                                <li>Ruby</li>
+                            </ul>
+                            <button type="button">See Project</button>
+                        </li>
+
+                        <li class="project">
+                            <h3>Profesional Art<br>Printing Data</h3>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                            <ul class="build-with">
+                                <li>html</li>
+                                <li>bootstrap</li>
+                                <li>Ruby</li>
+                            </ul>
+                            <button type="button">See Project</button>
+                        </li>
+
+                        <li class="project">
+                            <h3>Profesional Art<br>Printing Data</h3>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                            <ul class="build-with">
+                                <li>html</li>
+                                <li>bootstrap</li>
+                                <li>Ruby</li>
+                            </ul>
+                            <button type="button">See Project</button>
+                        </li>
+
+                        <li class="project">
+                            <h3>Profesional Art<br>Printing Data</h3>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                            <ul class="build-with">
+                                <li>html</li>
+                                <li>bootstrap</li>
+                                <li>Ruby</li>
+                            </ul>
+                            <button type="button">See Project</button>
+                        </li>
+                    </ul>
+                </div>
+            </section>
+        
+            <!--About Me Section-->
+            <section class="about">
+                <div class="about-me">
+                <h2>About me</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <button type="button" class="btn">Get My Resume</button>
```

Comment:
- [ ] kindly add the correct color for this button and try to apply the button styling: Active, Hover, and Pressed like shown in Figma

![image](https://user-images.githubusercontent.com/89282221/185761479-952ba3da-f9b3-4538-9e6d-adade252f92f.png)